config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -1203,8 +1203,11 @@ presubmits:
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200802-ff98b58-1.16
         name: ""
         resources:
+          limits:
+            cpu: 4
+            memory: 9000Mi
           requests:
-            cpu: "2"
+            cpu: 4
```

Comment:
what CPU are we giving the other bazel jobs currently?

Reply:
4 for the jobs that use RBE, I'm still trying to find something for the jobs that don't

Reply:
hmm ok
I'm nervous that limiting the CPU is going to cause build timeouts, but I guess I'm willing to try it